db/migrate/20170614135258_create_split_pdf_logs.rb

```diff
@@ -0,0 +1,22 @@
+class CreateSplitPdfLogs < ActiveRecord::Migration
+  def change
+    create_table :split_pdf_logs do |t|
+      t.string :split_pdf_console_log
+      t.string :error_description
+      t.string :uploaded_filetype, null: false
+      t.string :uploaded_filename, null: false
+      t.string :user, null: false
+      t.string :host, null: false
+      t.integer :number_of_new_files_in_raw_dir, null: false
+      t.integer :number_of_new_files_in_complete_dir, null: false
+      t.integer :number_of_new_files_in_incomplete_dir, null: false
+      t.integer :number_of_new_files_in_error_dir, null: false
+      t.integer :original_num_pages, null: false
+      t.integer :split_into_how_many_pages, null: false
```

Comment:
This is not necessary. However, a column indicating the number of pages that were errors in qr-code scanning is appropriate.

Not sure what "missing_pages" is below, but if it's what I imagine, I doubt it'll be useful. More important to know the fields above.